react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+export default class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isSatisfied: false };
+    this.handleClick = this.handleClick.bind(this);
+    this.modalOff = this.modalOff.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.handleClickWhenOn = this.handleClickWhenOn.bind(this);
+  }
+
+  handleClick() {
+    if (event.target.className === 'fas fa-bars fa-2x') {
+      this.setState(prevState => ({
+        isSatisfied: !prevState.isSatisfied
+      }));
+    }
+  }
+
+  handleClickWhenOn(e) {
+    if (event.target.className === 'modalOn' || event.target.className === 'choice') {
```

Comment:
If you are going to use `e` as the parameter name then you need to change the `event`s on this line to be `e`. The idea is you are using the parameter `e` and not the global `event`